sklearn/ensemble/_bagging.py

```diff
@@ -981,6 +982,14 @@ def decision_function(self, X):
 
         return decisions
 
+    def _more_tags(self):
+        if self.estimator is None:
+            estimator = DecisionTreeClassifier()
+        else:
+            estimator = self.estimator
```

Comment:
I am wondering if we should have something similar to the `available_if` where we delegate first to `estimator_` and then `estimator` if the estimator is not fitted.

I am thinking of the following case:
- fit a bagging estimator supporting `nan`
- set `estimator` to an estimator that does not support `nan`

At this stage, your model can still handle `nan` if `predict` is called.
I am not sure what is the expected behaviour of the tag there.

Reply:
Assuming `estimator_` **does** support nan and `estimator` does not support `nan`, then `bagging.fit` would not support `nan`. In this sense, `Bagging*` should get the `allow_nan` tag from `estimator`. Getting the tag from `estimator_` would give the incorrect tag.